python/GafferUI/LabelPlugValueWidget.py

```diff
@@ -65,6 +65,8 @@ def __init__( self, plug, horizontalAlignment=GafferUI.Label.HorizontalAlignment
 			formatter = self.__formatter,
 		)
 		self.__label._qtWidget().setObjectName( "gafferPlugLabel" )
+		if any( p.direction() == Gaffer.Plug.Direction.In for p in self.getPlugs() ) :
+			self.__label._qtWidget().setProperty( "gafferValueChanged", GafferUI._Variant.toVariant( False ) )
```

Comment:
Can we do this in `__setValueChanged()` instead of here, so that everything is all in one place?

Reply:
I have a potential change for this in https://github.com/GafferHQ/gaffer/pull/5737/commits/61ebf0e67e906c22f4cd557de90163b8f9bc02de. It's nice to get that all in one place, but it does require a `setStyleSheet()` call in order to recognized the `padding-left` change.

It doesn't seem to have caused jumpy layouts or other issues, but I don't know if there are knock-on effects from `setStyleSheet()`.